### PR TITLE
chore: release Ruby SDK

### PR DESCRIPTION
**Pull request by armandobelardo**

### CODE REVIEW

File: generators/ruby/sdk/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [0.0.6] - 2024-02-07
+- Improvement: license files are now specified within the gem config if they are provided
```
Comment (dsinghvi) [inline generators/ruby/sdk/CHANGELOG.md:11]: can we add a small example of the config in `generators.yml`

File: generators/ruby/sdk/CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Improvement: we now create a Github workflow file for publishing the gem automatically, in the presence of a `publishInfo` configuration
```
Comment (dsinghvi) [inline generators/ruby/sdk/CHANGELOG.md:12]: Users don't know about `publishInfo`, can we add an example of the `generators.yml` configuration with the `output` key

File: generators/ruby/sdk/CHANGELOG.md (hunk 1 above)
Comment (armandobelardo) [reply generators/ruby/sdk/CHANGELOG.md:11]: it's not ruby specific though?

File: generators/ruby/sdk/CHANGELOG.md (hunk 2 above)
Comment (armandobelardo) [reply generators/ruby/sdk/CHANGELOG.md:12]: Is this meant to be consumed by users?

Comment (dsinghvi) [reply generators/ruby/sdk/CHANGELOG.md:12]:
```yaml
generators: 
  - name: 
     output: 
       location: rubygems
       ...
```

File: generators/ruby/sdk/CHANGELOG.md (hunk 1 above)
Comment (dsinghvi) [reply generators/ruby/sdk/CHANGELOG.md:11]: It's not, but we haven't documented it well enough so anyone who reads this changelog should be able to take advantage of it is my thought

File: generators/ruby/sdk/CHANGELOG.md (hunk 2 above)
Comment (dsinghvi) [reply generators/ruby/sdk/CHANGELOG.md:12]: Yes it is

**Review by dsinghvi — CHANGES_REQUESTED** [review]
need to submit a comment to finish review

**Review by dsinghvi — APPROVED** [review]
g2g once the yaml in the CHANGELOG are valid!

**Pull request merged**